app/controllers/application_controller.rb

```diff
@@ -11,6 +11,16 @@ def current_user
   end
   helper_method :current_user
 
+  def logged_in?
+    session[:user_id].present?
+  end
```

Comment:
Is this method really needed? or could you use the `current_user` method?

Reply:
Duh, thanks, done in sha: ae72639